### PR TITLE
Module 5

### PR DESCRIPTION
**Pull request by josephsilcock**
Add docker things for prod & dev builds

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    build: .
+    ports:
+      - 5000:5000
+    env_file: ./.env
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /todo-app/todo_app
+    image: todo-app:dev
+  todo-app-test:
```
Comment (josephsilcock) [inline docker-compose.yml:13]: This doesn't seem to actually work...not sure why :(

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  todo-app-dev:
+    build: .
+    ports:
+      - 5000:5000
+    env_file: ./.env
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /todo-app/todo_app
+    image: todo-app:dev
```
Comment (BenRamchandani) [inline docker-compose.yml]: You don't need both `image` and `build`, one is enough. You also shouldn't rely on `development` being the last target in your dockerfile, you can specify which to build:

```yaml
build:
  context: .
  target: development
```

File: docker-compose.yml (hunk 1 above)
Comment (BenRamchandani) [reply docker-compose.yml:13]: Yeah `watchmedo` doesn't seem to work :(
It should be `poetry run watchmedo` really, but that doesn't help either, it doesn't pick up any changes.

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+scripts/
+tests/
+.env
+.env.j2
+.env.template
+.env.test
+ansible-inventory
+ansible-playbook.yml
+setup.py
+todoapp.service.j2
```
Comment (BenRamchandani) [inline .dockerignore:14]: Could add `**/__pycache__/`

**Review by BenRamchandani — APPROVED** [review]
Looks great, I can get it running in Docker using the instructions in your readme. Go ahead and merge when you're ready.

**Pull request merged**